tests/mock_crust/utils.rs

```diff
@@ -62,6 +62,14 @@ pub fn gen_range_except<T: Rng>(
     x
 }
 
+fn create_config(network: &Network<PublicId>) -> Config {
+    Config {
+        dev: Some(DevConfig {
+            min_section_size: Some(network.min_section_size()),
+            ..DevConfig::default()
```

Comment:
Means just fill in the rest of the uninitialised fields with those from the thing after the `..`.